config/config.go

```diff
@@ -31,17 +31,21 @@ func NewDefaultConfig() *AgentConfig {
 
 		Admin: AdminConfig{
 			Auth: ServiceAuthConfig{
-				Clients:     make([]OAuthClientCredentials, 0),
-				HMACSecrets: make([]string, 0),
-				TTL:         0,
+				Clients:            make([]OAuthClientCredentials, 0),
+				HMACSecrets:        make([]string, 0),
+				TTL:                0,
+				JwksURL:            "",
+				JwksUpdateInterval: 0,
 			},
 			Port: "8088",
 		},
 		API: APIConfig{
 			Auth: ServiceAuthConfig{
-				Clients:     make([]OAuthClientCredentials, 0),
-				HMACSecrets: make([]string, 0),
-				TTL:         0,
+				Clients:            make([]OAuthClientCredentials, 0),
+				HMACSecrets:        make([]string, 0),
+				TTL:                0,
+				JwksURL:            "",
+				JwksUpdateInterval: 0,
```

Comment:
Could you add JWKS update interval to the table of config parameters in the main README?